### DISCUSSION TITLE
How to show more result?

### DISCUSSION THREAD

**Discussion by iimka** [kernc/backtesting.py#927] [General]
I used the code: 
print(result['_trades'])
but only few results were shown. How to show more?

![image](https://user-images.githubusercontent.com/55306556/222875480-1040d809-81b5-4ee9-9178-20634c4a82c4.png)

**Comment by kernc** [comment]
Results stats/trades are pandas DataFrames, therefore they respect [pandas display options](https://pandas.pydata.org/docs/user_guide/options.html).
